htlcswitch/decayedlog.go

```diff
@@ -41,6 +41,35 @@ var (
 	ErrDecayedLogCorrupted = errors.New("decayed log structure corrupted")
 )
 
+// NewBoltBackendCreator returns a function that creates a new bbolt backend for
+// the decayed logs database.
+func NewBoltBackendCreator(dbPath,
```

Comment:
We could move this function under the `kvdb` module to reduce duplication.

Reply:
My main concern was to not create more package dependencies between `lncfg` and `macaroons`/`watchtower`. But since we move all other DB related stuff to `lncfg` I decided to also move the database filename constants there to avoid those dependencies.